data-serving/data-service/src/util/search.ts

```diff
@@ -1,16 +1,40 @@
 import { parse, SearchParserResult } from 'search-query-parser';
+import { resolveConfigFile } from 'prettier';
 
 export interface ParsedSearch {
     fullTextSearch?: string;
+    filters: {
+        path: string;
+        values: string[];
+    }[];
 }
 
+// Map of keywords to their case data path.
+// IMPORTANT: If you change this mapping, reflect the new keys in the openapi.yaml file as well.
+const keywords = new Map<string, string>([
+    ['curator', 'revisionMetadata.creationMetadata.curator'],
```

Comment:
I wonder if people will expect their edits to come up as well when searching for curator

Reply:
hmm, yeah. we could add an "editor" keyword that would match if we ever get that FR.